x509_build/install.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
`/bin/sh` will suffice here, some systems will lack bash or have it at another location.